plugins/v8-xhr.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Chrome/V8 XHR Post plugin
+ *
+ * Allows raven-js to send log entries using XHR and CORS for
+ * environments where the Origin and Referer headers are missing (e.g Chrome plugins).
+ *
+ * Usage: Raven.config('https+post://...');
+ * */
+
+;(function(window, Raven, undefined){
+    'use strict';
+    var V8Transport = window.V8Transport = {
```

Comment:
Because sadly, this doesn't just work everywhere that XHR is accepted, so this is specific to V8.